SwapMeter.c

```diff
@@ -38,6 +39,9 @@ static void SwapMeter_display(const Object* cast, RichString* out) {
    Meter_humanUnit(buffer, this->values[0], sizeof(buffer));
    RichString_appendAscii(out, CRT_colors[METER_TEXT], " used:");
    RichString_appendAscii(out, CRT_colors[METER_VALUE], buffer);
+   Meter_humanUnit(buffer, this->values[1], sizeof(buffer));
```

Comment:
Please a blank line.

Also this block should only be compiled on Linux (`#ifdef HTOP_LINUX`), as on other platforms it will always show `0K`.